vault/identity_store_oidc_provider_test.go

```diff
@@ -712,6 +810,38 @@ func TestOIDC_Path_OIDC_Authorize(t *testing.T) {
 			},
 			wantErr: ErrAuthInvalidRequest,
 		},
+		{
```

Comment:
Can we also add a test case for the following scenario?

code_challenge | code_challenge_method | behavior
-- | -- | --
Present | Absent | PKCE used with default method from rfc7636 (plain)

Reply:
Done in 2815f9b 👍